pyproject.toml

```diff
@@ -15,9 +15,14 @@ profile = "black"
 
 [tool.mypy]
 files = "src"
-python_version = "3.6"
+exclude = [
+    'decaylanguage\.modeling\.*',
```

Comment:
@henryiii, do you happen to know why the checking is being done on the modeling submodule as well when I'm trying here to ignore it? This line  hence isn't correct, somehow.

Reply:
This is not a valid regular expression (the star at then end is not preceded by anything to have 0 or more of). I think it should be

```suggestion
    '^decaylanguage/modeling',
```

It may need to be in an overrides too if it's being imported from files you are checking. If you are excluding the file from being an input file, but it's still being imported somewhere, it's still being checked, at least partially. You may want:

```toml
[[tool.mypy.overrides]]
module = "decaylanguage.modeling.*"
# Disable whatever it's struggling with here
```

Final]y, pre-commit might be passing these files by name if you haven't excluded them there.

Reply:
Let's see, thanks.